src/main/kotlin/io/github/inductiveautomation/kindling/thread/model/ThreadModel.kt

```diff
@@ -67,6 +67,21 @@ sealed class ThreadColumnList : ColumnList<ThreadLifespan>() {
         },
     )
 
+    val totalCPU = Column<ThreadLifespan, Double?>(
```

Comment:
Accidentally pulled in from another PR?

Reply:
Looks like it. We'll remove it from here.